docs/about/neon_ecosystem.md

```diff
@@ -98,5 +84,27 @@ Oracles enable the decentralized Web3 ecosystem to access price feeds and other
 
 ### Wallets
 
-Neon EVM is supported by the MetaMask non-custodial wallet.
+Neon EVM is supported by the MetaMask non-custodial wallet. 
+
+> This allows dApp devs to sign deployment transactions and mint tokens. 
+<!-- > Learn more about minting and wrapping tokens with Neon EVM. suggest adding link here as Phantom wallet is invovled for Solana-based mints -->
+
+### Protocols
```

Comment:
let's move it to Wallets

Reply:
Not sure what you want moved @YYYurchenko